src/test/kotlin/com/ajax/cryptocurrency/grpc/CryptocurrencyGrpcServiceTest.kt

```diff
@@ -40,13 +42,15 @@ class CryptocurrencyGrpcServiceTest(
 ) {
 
     @MockK
-    private lateinit var cryptocurrencyRepository: CryptocurrencyRepositoryOutPortImpl
+    private lateinit var cryptocurrencyRepository: CryptocurrencyRepositoryOutPort
```

Comment:
using of 
```
@SpringBootTest
@ExtendWith(MockKExtension::class)
```
in the same test class looks really wierd. I bet you should choose only one.

Reply:
we can setup a call if you want to discuss